### ISSUE TITLE
literate cannot find module

### ISSUE THREAD

**Issue by theonlygusti** [theonlygusti/befunge#2]
```
module.js:472
    throw err;
    ^

Error: Cannot find module '../.././versions/Befunge-93'
    at Function.Module._resolveFilename (module.js:470:15)
    at Function.Module._load (module.js:418:25)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at parseJS (../node_modules/literate/index.js:30:9)
    at Object.<anonymous> (./befunge.js:2:15)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
```

**Comment by theonlygusti** [comment]
This error is caused by [the '93 branch](/theonlygusti/befunge/tree/'93) since commit ba5842b0575de3adf656280b9b68badcfd5a988f

**Comment by theonlygusti** [comment]
Fixed by 9f2d365a4ec54c788b292d43cdf9b421cf3c328e
